201-vmss-azureautomationdsc/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "VM Scale Set Configuration managed by Azure Automation",
+  "description": "Deploy a VM Scale Set where virtual machines are deployed as registered nodes in the Azure Automation Desired State Configuration service, and node configuration is guaranteed consistency after deployment. NOTE: The DSC configuration module requires three specific Azure Automation DSC parameters: Registration Key, Registration URL, and Configuration Function. These prerequisites are available only after successful creation and configuration of an Azure Automation Account for Azure Automation DSC.",
```

Comment:
Maybe just say the reg key / url of an existing Azure Automation account are required.